Automattic/jetpack
# Adapt Jetpack to use WordPress JS i18n, fix Gutenberg block i18n.

#### Changes proposed in this Pull Request:
* Adapts JSON language files to be usable in WordPress client-side i18n.
* Rebuilds JSON language files.
* Adds a mechanism to integrate POT files that result from [block building](https://github.com/Automattic/wp-calypso/pull/30303) into `jetpack-strings.php`. The whole PHP file generation pipeline now consists of one i18n-calypso run, one msgcat run, and one msgexec run that executes a bash script for every translation.
* Paves the way for legacy i18n support and makes initial steps to move towards wp.i18n.

#### Testing instructions:
Currently blocks don't have any strings translated yet in the JSON files because the strings themselves are not present in the code. After merging this code we should be able to re-run language builds and get translated strings in blocks.
* Make sure the existing i18n works properly, including existing wp-admin Settings translations.

#### Proposed changelog entry for your changes:
* Adapted existing Jetpack translations to be used in WordPress i18n mechanisms.
